doc/source/management/data_at_rest_encryption.rst

```diff
@@ -166,12 +138,12 @@ encryption of system temporary tablespace.
   :vartype: Text
   :default: ``OFF``
 
-This feature is considered **BETA** quality.
-
-This variable has 3 possible values. ``ON`` makes |InnoDB| tables encrypted by
+This variable has 5 possible values. ``ON`` makes |InnoDB| tables encrypted by
```

Comment:
5 possible, but 3 given in this paragraph

Reply:
Also, I think the new 2 values should be alpha, not beta.

Reply:
I do not see all the new values ONLINE_TO_KEYRING etc

Reply:
In whole docs there is no word about encryption threads

Reply:
Or default_encryption_key_id

Reply:
This default_encryption_key_id can be altered on tablespace level with ALTER TABLE (…) ENCRYPTION=’KEYRING’ ENCRYPTION_KEY_ID=X.

Reply:
what about innodb_encryption_threads and all the other variables ?

Reply:
@robgolebiowski ONLINE_TO_KEYRING and ONLINE_TO_KEYRING_FORCE are these values also *Alpha*?

Reply:
These two values are also alpha

Reply:
The requested values have been added